src/org/traccar/api/resource/NotificationResource.java

```diff
@@ -49,7 +49,7 @@
             userId = getUserId();
         }
         Context.getPermissionsManager().checkUser(getUserId(), userId);
-        return Context.getNotificationManager().getUserNotifications(userId);
+        return Context.getNotificationManager().getAllUserNotifications(userId);
```

Comment:
Do we still need "all" parameter in this method? Maybe we can also remove `getAllNotifications` or make it private?